std/container/util.d

```diff
@@ -140,3 +140,37 @@ unittest
     auto rbtmin = make!(RedBlackTree, "a < b", false)(3, 2, 2, 1);
     assert(equal(rbtmin[], [1, 2, 3]));
 }
+
+package:
+/**
+ *String representation of the container
+ */
+mixin template forwardToStringTo(alias symbol)
+{
+    string toString() {
```

Comment:
```d
void toString(scope void delegate(const(char)[]) sink, FormatSpec!char fmt)
{
    import std.format : formatValue;
    import std.internal.scopebuffer : scopeBuffer;
    char[256] buffer; // arbitrary size
    auto sBuffer = scopeBuffer(buffer[]);
    formatValue(sBuffer, symbol, fmt);
    sink(sBuffer[]);
}
```
It might be prudent to wrap the usage of `ScopeBuffer` in `@trusted`.

A better implementation might be to iterate `this[]` and calling the sink with the stringified result of each element, although this would need to take custom `toString` (on the range type) and such into account, so this logic probably belongs somewhere in std.conv or std.format, not anywhere in the std.container package.

See also [the wiki page for sink-based toString](http://wiki.dlang.org/Defining_custom_print_format_specifiers).

(Note also the brace style; this style is the Phobos convention)